scipy/stats/unuran/src/THANKS

```diff
@@ -0,0 +1,20 @@
+We thank the following people for their permissions to use
+some of their codes:
+
+Ernst Stadlober (Stadlober@stat.tu-graz.ac.at)
+   ... WinRand library. 
+
+Renee Touzin 
+   ... mrg31k3p.c  (Combined multiple recursive generator)
```

Comment:
we could remove / add a comment here

Reply:
not relevant for this PR, we need to check in the submodule that contains the UNU.RAN code later